packages/create-react-app/createReactApp.js

```diff
@@ -77,6 +77,7 @@ const program = new commander.Command(packageJson.name)
   )
   .option('--use-npm')
   .option('--use-pnp')
+  .option('--modern')
```

Comment:
We shouldn't need to change this? This is for the `create-react-app` cli.

Reply:
Oh I'm sorry I'm not too familiar with this part